assets/js/googlesitekit/datastore/forms/index.test.js

```diff
@@ -19,73 +19,111 @@
 /**
  * Internal dependencies
  */
-import {
-	createTestRegistry,
-	unsubscribeFromAll,
-} from 'tests/js/utils';
+import { createTestRegistry } from 'tests/js/utils';
 import { STORE_NAME } from './constants';
 
 describe( 'core/forms store', () => {
-	const formName = 'Form 1';
-	const formData = {
-		key1: 'value1',
-		key2: 'value2',
-	};
-	const newFormName = 'Form 2';
-	const newFormData = {
-		key3: 'value3',
-		key4: 'value4',
-	};
 	let registry;
 
 	beforeEach( () => {
 		registry = createTestRegistry();
 	} );
 
-	afterEach( () => {
-		unsubscribeFromAll( registry );
-	} );
-
 	describe( 'actions', () => {
 		describe( 'setValues', () => {
 			it( 'requires the formName param', () => {
 				expect( () => {
 					registry.dispatch( STORE_NAME ).setValues();
 				} ).toThrow( 'formName is required for setting values.' );
 			} );
+
 			it( 'requires the formData param', () => {
+				const formName = 'Form 1';
```

Comment:
Extracting the form name to a variable is a bit necessary in most cases here. It would be more clear to use a self-describing string literal like "form-name".

Minor nitpick: let's avoid using numbered names unless the test is uses multiple forms. I find using letters for generic names for multiple things, and numbers only when order is significant to be a good rule of thumb that helps improve readability.